raiden/tests/benchmark/profile_transfer.py

```diff
@@ -183,21 +191,37 @@ def profile_transfer(num_nodes=10, channels_per_node=2):
         for position in range(num_nodes)
     ]
 
-    BlockChainServiceMock.reset()
     blockchain_services = list()
+    tester = tester_state(
```

Comment:
This should be done in another pr, this one is only removing the mock implementation.

BTW, the benchmark scripts are not working, since they are not executed in the build and have falled behind.